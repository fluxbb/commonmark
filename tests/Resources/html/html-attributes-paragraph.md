{#id} Lorem ipsum dolor sit amet

{#id} Lorem ipsum dolor sit amet,
consectetur adipisicing elit.
Qui dicta minus molestiae vel
beatae natus eveniet ratione temporibus

{#id}
Lorem ipsum dolor sit amet,
consectetur adipisicing elit.
Qui dicta minus molestiae vel
beatae natus eveniet ratione temporibus

Lorem ipsum dolor sit amet,
consectetur adipisicing elit.
Qui dicta minus molestiae vel
beatae natus eveniet ratione temporibus {#id}

Lorem ipsum dolor sit amet,
consectetur adipisicing elit.
Qui dicta minus molestiae vel
beatae natus eveniet ratione temporibus
{#id}