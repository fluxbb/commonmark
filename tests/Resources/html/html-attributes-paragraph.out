<p id="id">Lorem ipsum dolor sit amet</p>

<p id="id">Lorem ipsum dolor sit amet,
consectetur adipisicing elit.
Qui dicta minus molestiae vel
beatae natus eveniet ratione temporibus</p>

<p id="id">Lorem ipsum dolor sit amet,
consectetur adipisicing elit.
Qui dicta minus molestiae vel
beatae natus eveniet ratione temporibus</p>

<p id="id">Lorem ipsum dolor sit amet,
consectetur adipisicing elit.
Qui dicta minus molestiae vel
beatae natus eveniet ratione temporibus</p>

<p id="id">Lorem ipsum dolor sit amet,
consectetur adipisicing elit.
Qui dicta minus molestiae vel
beatae natus eveniet ratione temporibus</p>